### PR TITLE
Laf 250/update category pages

### PR DESCRIPTION
**Pull request by hermanator608**

### CODE REVIEW

File: client/src/Admin/Category/CategoryModify.js [hunk 1]
```diff
@@ -15,121 +19,160 @@ class CategoryModify extends Component {
...
-        let sub = this.props.data.categories.find((sub) => e.id === sub._id);
-        if (sub && sub.parent) {
-          sub.new_parent = this.props.category;
-          if (sub.new_parent._id !== sub.parent._id) {
-            let i = sub.parent.subcategories.indexOf(sub._id);
-            sub.parent.subcategories.splice(i, 1);
-          }
-          full_subs.push(sub);
-        }
+
```
Comment (triqp) [inline client/src/Admin/Category/CategoryModify.js:55]: Are these comments necessary?

Comment (hermanator608) [reply client/src/Admin/Category/CategoryModify.js:55]: Good catch!

**Review by triqp — APPROVED** [review]

**Review by wozniakbe — APPROVED** [review]

**Review by wozniakbe — CHANGES_REQUESTED** [review]
Lots of scripts such as "npm start" got deleted, any reason?

**Comment by triqp** [comment]
Not needed since we don’t actually have a server to start. It can be added back if you really want it

**Review by wozniakbe — APPROVED** [review]
Tripp explained why scripts were removed, looks good

**Review by muencha — APPROVED** [review]

**Pull request merged**